README.md

```diff
@@ -1,3 +1,5 @@
+# **DEPRECATED** - New releases (`>= 3.0.0`) are only available in the new [spec-repo](https://github.com/bitmovin/cocoapod-specs)!
```

Comment:
Maybe worth adding a note that also v2 releases are published to the new repo? Otherwise customers might understand this as they have to use this repo for v2 and the new one for v3 although they could just go with the new one immediately.

Reply:
Good point, what do you think about this? 🤔 

---

# **DEPRECATED** - This repository is deprecate in favour of our new [spec-repo](https://github.com/bitmovin/cocoapod-specs)! 

## The new repository contains all `v2` versions which are available in this repository, as well as additional releases of the new major version (`3.0.0`).

Reply:
LGTM 👍 
Small typo: should be `deprecated` (missing `d`)

Reply:
fc69855